var/spack/repos/builtin/packages/py-pydv/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPydv(PythonPackage):
+    """PDV is a 1D graphics and data analysis tool, heavily based on the
+    ULTRA plotting tool"""
+
+    homepage = "https://github.com/griffin28/PyDV"
+    url      = "https://github.com/griffin28/PyDV.git"
+
+    version('2.4.2', git=url, tag='pydv-2.4.2')
```

Comment:
Is there a reason you're not using the tarball from the releases page?

Reply:
Whoops, didn't realize that was there; retooling ...

Reply:
Pushed a change to use the tarball from the releases page; all CI checks passed - resolving